Module doesn't recover when Immich is unavailable for a moment

When Immich is unavailable while the module is communicating with it, it doesn't handle the exception and I need to restart MagicMirror for it to start working again.

```
magicmirror-1  | [2026-03-09 09:49:13.179] [ERROR] [MMM-ImmichSlideShow] MMM-ImmichSlideShow :: immichApi :: Oops!  Exception while fetching server version connect ECONNREFUSED 192.168.1.242:2283
magicmirror-1  | [2026-03-09 09:49:13.180] [ERROR] [app] Whoops! There was an uncaught exception...
magicmirror-1  | [2026-03-09 09:49:13.180] [ERROR] [app] UnhandledPromiseRejection: This error originated either by throwing inside of an
async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason
"Failed to get Immich version.  Cannot proceed.".
magicmirror-1  |     at throwUnhandledRejectionsMode (node:internal/process/promises:392:7)
magicmirror-1  |     at processPromiseRejections (node:internal/process/promises:475:17)
magicmirror-1  |     at process.processTicksAndRejections (node:internal/process/task_queues:104:32) {
magicmirror-1  |   code: 'ERR_UNHANDLED_REJECTION'
magicmirror-1  | }
magicmirror-1  | [2026-03-09 09:49:13.181] [ERROR] [app] MagicMirror² will not quit, but it might be a good idea to check why this happened. Maybe no internet connection?
magicmirror-1  | [2026-03-09 09:49:13.181] [ERROR] [app] If you think this really is an issue, please open an issue on GitHub: https://github.com/MagicMirrorOrg/MagicMirror/issues
```

--- a/node_helper.js
+++ b/node_helper.js
@@ -138,8 +138,13 @@ module.exports = NodeHelper.create({
       return;
     }
     Log.debug(LOG_PREFIX + 'GOT new active config', this.config.activeImmichConfig);
-    await immichApi.init(config.activeImmichConfig, this.expressApp, isActiveConfigChange);
-   
+    try {
+      await immichApi.init(config.activeImmichConfig, this.expressApp, isActiveConfigChange);
+    } catch (e) {
+      Log.warn(LOG_PREFIX + 'Immich is not available or failed to get version.  Returninging without the image list');
+      return;
+    }
+    
     // create an empty main image list
     this.imageList = [];
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "mmm-immichslideshow",
-  "version": "1.7.0",
+  "version": "1.7.1",
   "description": "A plugin for MagicMirror2 to show slideshows from Immich",
   "author": "Pelaxa Inc.",
   "license": "MIT",

